pkg/routing/selector/cpuload.go

```diff
@@ -31,12 +29,11 @@ func (s *CPULoadSelector) filterNodes(nodes []*livekit.Node) ([]*livekit.Node, e
 	return nodes, nil
 }
 
-func (s *CPULoadSelector) SelectNode(nodes []*livekit.Node) (*livekit.Node, error) {
+func (s *CPULoadSelector) SelectNode(nodes []*livekit.Node, sortBy string) (*livekit.Node, error) {
```

Comment:
instead of changing `SelectNode`, IMO it's cleaner to have sortby as a field in the node selector itself during construction.

Reply:
I can do that if you prefer. I thought this seemed cleaner since it's really only needed in the SelectNode call, but I can do an update to move it to the selector constructor.

Reply:
that would be great 🙏  I'd like to keep the `SelectNode` interface so that it only needs to take in `nodes`. Especially if we started adding other options that are specific to certain selectors.